documentation/staging/content/userguide/managing-domains/domain-lifecycle/startup.md

```diff
@@ -207,15 +207,27 @@ The server will count toward the cluster's `replicas` count.  Also, if you confi
 The Domain YAML file includes the field `serverPod` that is available under `spec`, `adminServer`, and each entry of
 `clusters` and `managedServers`. The `serverPod` field controls many details of how Pods are generated for WebLogic Server instances.
 
-The `shutdown` field of `serverPod` controls how servers will be shut down and has three fields:
-`shutdownType`, `timeoutSeconds`, and `ignoreSessions`.  The `shutdownType` field can be set to either `Graceful`, the default,
-or `Forced` specifying the type of shutdown.  The `timeoutSeconds` property configures how long the server is given to
-complete shutdown before the server is killed.  The `ignoreSessions` property, which is only applicable for graceful shutdown, when `false`,
-the default, allows the shutdown process to take longer to give time for any active sessions to complete up to the configured timeout.
-The operator runtime monitors this property but will not restart any server pods solely to adjust the shutdown options.
+The `shutdown` field of `serverPod` controls how managed servers will be shut down and has the following four properties:
+`shutdownType`, `timeoutSeconds`, `ignoreSessions` and `waitForAllSessions`. The operator runtime monitors these properties but will not restart any server pods solely to adjust the shutdown options.
 Instead, server pods created or restarted because of another property change will be configured to shutdown, at the appropriate
 time, using the shutdown options set when the WebLogic Server instance Pod is created.
 
+| Field| Default Value | Supported Values | Description |
+| --- | --- | --- | --- |
+| `shutdownType` | `Graceful` | `Graceful` or `Forced` | Specifies how the operator will shut down server instances. |
+| `timeoutSeconds` | 30 | Whole number in seconds where 0 means no timeout | For graceful shutdown only, number of seconds to wait before aborting in-flight work and shutting down the server. |
```

Comment:
no timeout. (period at the end of the sentence)

Reply:
added period